etl/resources/wikipedia_reader_resource.py

```diff
@@ -0,0 +1,39 @@
+import json
+from collections.abc import Iterable
+from pathlib import Path
+
+from dagster import ConfigurableResource
+from unidecode import unidecode
+
+from etl.models import wikipedia
+from etl.models.types import DataFileName
+
+
+class WikipediaReaderResource(ConfigurableResource):  # type: ignore
+    """A ConfigurableResource to read Wikipedia data from storage."""
+
+    data_file_names: list[DataFileName]
+
+    def read(self) -> Iterable[wikipedia.Article]:
+        """Read in Wikipedia data and yield them as wikipedia.Article objects."""
+
+        for data_file_name in self.data_file_names:
+
+            data_file_path = Path(__file__).parent.parent / "data" / data_file_name
```

Comment:
You're hard-coding these paths everywhere.

Reply:
I am using the `DataFilesConfig` pattern now.